tests/protocol/test_extensions_goal_state.py

```diff
@@ -28,6 +29,32 @@ def test_property(name, value):
             test_property("required_features",       ['MOCK_REQUIRED_FEATURE'])
             test_property("on_hold",                 True)
 
+    def test_compare_should_check_the_status_upload_blob_only_when_the_host_ga_plugin_version_is_greater_then_112(self):
+        with mock_wire_protocol(mockwiredata.DATA_FILE_VM_SETTINGS) as protocol:
+            from_extensions_config = protocol.client.get_extensions_goal_state()
+            from_vm_settings = protocol.client._extensions_goal_state_from_vm_settings
+
+            # we set a the status upload value to a dummy value to verify whether the compare() method is checking it or not
+            from_vm_settings._status_upload_blob = "A DUMMY VALUE FOR THE STATUS UPLOAD BLOB"
+
+            #
+            # 112 and below should not check the status blob
+            #
+            from_vm_settings._host_ga_plugin_version = FlexibleVersion("1.0.8.112")
```

Comment:
Maybe worth adding a check where we dont get any HostGA version at all too?

Reply:
didn't consider it interesting, since flexible version inits to 0